db/config.cc

```diff
@@ -416,16 +416,16 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     */
     , commit_failure_policy(this, "commit_failure_policy", value_status::Unused, "stop",
         "Policy for commit disk failures:\n"
-        "* die          Shut down gossip and Thrift and kill the JVM, so the node can be replaced.\n"
-        "* stop         Shut down gossip and Thrift, leaving the node effectively dead, but can be inspected using JMX.\n"
+        "* die          Shut down gossip, so the node can be replaced.\n"
+        "* stop         Shut down gossip, leaving the node effectively dead, but can be inspected using the RESTful APIs.\n"
         "* stop_commit  Shut down the commit log, letting writes collect but continuing to service reads (as in pre-2.0.5 Cassandra).\n"
         "* ignore       Ignore fatal errors and let the batches fail."
         , {"die", "stop", "stop_commit", "ignore"})
     , disk_failure_policy(this, "disk_failure_policy", value_status::Unused, "stop",
         "Sets how Scylla responds to disk failure. Recommend settings are stop or best_effort.\n"
-        "* die              Shut down gossip and Thrift and kill the JVM for any file system errors or single SSTable errors, so the node can be replaced.\n"
-        "* stop_paranoid    Shut down gossip and Thrift even for single SSTable errors.\n"
-        "* stop             Shut down gossip and Thrift, leaving the node effectively dead, but available for inspection using JMX.\n"
+        "* die              Shut down gossip for any file system errors or single SSTable errors, so the node can be replaced.\n"
+        "* stop_paranoid    Shut down gossip even for single SSTable errors.\n"
+        "* stop             Shut down gossip, leaving the node effectively dead, but available for inspection using the RESTful APIs.\n"
```

Comment:
I think there may be a mistake in this silly message, but I suspect it's not used (just look at its old content ;-)) so perhaps who cares... The message suggests that "stop" leaves the node effectively dead by shutting down gossip and thrift (which you now removed), but what about CQL??? Doesn't it shut down CQL too?

Anyway, since this entire message looks like outdated crap, I don't insist you fix it now. But maybe if you feel like and already touching this, you can investigate what this configuration really does.

Reply:
tl;dr,  the answer is 

> it does nothing.

but i believe it's on purpose, as it's marked `Unused`. so, i think the reason why it's here is that Cassandra has it, and we want to be compatible with Cassandra. and the "Unused" here matches with its document at https://github.com/scylladb/scylladb/blob/8a72324ff1ed3509e5f59d8ab26da0c7f413be0f/utils/config_file.hh#L72-L81 . so i'd like to keep as it is.